agent/consul/leader.go

```diff
@@ -349,32 +348,48 @@ func (s *Server) reconcileReaped(known map[string]struct{}) error {
 			continue
 		}
 
-		// Create a fake member
-		member := serf.Member{
-			Name: check.Node,
-			Tags: map[string]string{
-				"dc":   s.config.Datacenter,
-				"role": "node",
-			},
-		}
-
 		// Get the node services, look for ConsulServiceID
 		_, services, err := state.NodeServices(nil, check.Node)
 		if err != nil {
 			return err
 		}
 		serverPort := 0
+		serverAddr := ""
+		serverID := ""
+
 		for _, service := range services.Services {
 			if service.ID == structs.ConsulServiceID {
 				serverPort = service.Port
+				_, node, err := state.GetNode(check.Node)
+				if err != nil {
+					s.logger.Printf("[ERR] Unable to look up node with name %v, got error:%v", check.Node, err)
```

Comment:
Should be `[ERR] consul: ` for the prefix, and we usually just do a colon before the error, so `[ERR] consul: Unable to look up node with name %q: %v"`